INDEX.md

```diff
@@ -0,0 +1,84 @@
+| 2020 | [Marking APIs that are unsupported by Blazor WebAssembly](accepted/2020/blazor-unsupported-apis/blazor-unsupported-apis.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2020 | [Marking Windows-only APIs](accepted/2020/windows-specific-apis/windows-specific-apis.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2020 | [OR_GREATER preprocessor symbols for TFMs](accepted/2020/or-greater-defines/or-greater-defines.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2020 | [Obsoletions in .NET 5](accepted/2020/better-obsoletion/obsoletions-in-net5.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2020 | [Overview and goals](accepted/2020/serializer/SerializerGoals5.0.md) | [Steve Harter](https://github.com/steveharter) |
+| 2020 | [Platform Specific HttpClient Support in dotnet/runtime](accepted/2020/mono-convergence/platform-specific-httpclient.md) | [Steve Pfister](https://github.com/steveisok) |
+| 2020 | [Portable PDB](accepted/2020/diagnostics/portable-pdb.md) | [Tomáš Matoušek](https://github.com/tmat) |
+| 2020 | [Proposal for global.json](accepted/2020/global-json-updates.md) | [Kathleen Dollard](https://github.com/KathleenDollard) |
+| 2020 | [Publishing and Consuming Symbols and Source for Debugging](accepted/2020/diagnostics/debugging-with-symbols-and-sources.md) | [Tomáš Matoušek](https://github.com/tmat) |
+| 2020 | [Reproducible Build Validation](accepted/2020/reproducible-builds.md) | [Claire Novotny](https://github.com/clairernovotny), [Jared Parsons](https://github.com/jaredpar) |
+| 2020 | [Single-file Publish](accepted/2020/single-file/design.md) | [Swaroop Sridhar](https://github.com/swaroop-sridhar) |
+| 2020 | [Single-file Publish for .NET Core 3.0](accepted/2020/single-file/design_3_0.md) | [Swaroop Sridhar](https://github.com/swaroop-sridhar) |
+| 2020 | [Source Link](accepted/2020/diagnostics/source-link.md) | [Tomáš Matoušek](https://github.com/tmat) |
+| 2020 | [Target Framework Names in .NET 5](accepted/2020/net5/net5.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2020 | [Type for holding & converting binary data](accepted/2020/binary-data/binary-data.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2021 | [.NET 6.0 Target Frameworks](accepted/2021/net6.0-tfms/net6.0-tfms.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2021 | [Designs](accepted/2021/statics-in-interfaces/designs/README.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2021 | [Requirements](accepted/2021/statics-in-interfaces/requirements/README.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2021 | [Scenarios](accepted/2021/statics-in-interfaces/scenarios/README.md) | [Immo Landwerth](https://github.com/terrajobst) |
+| 2021 | [Statics in Interfaces](accepted/2021/statics-in-interfaces/README.md) | [Immo Landwerth](https://github.com/terrajobst) |
```

Comment:
Static in interfaces are split over multiple documents. It is not clear by looking at the index that all these 4 documents are for static in interfaces.

Reply:
Thanks for catching this! Do you think it makes more sense to have a single entry for "Statics In Interfaces" or should we list the individual sub-designs with a prefix like "Statics In Interfaces"?

Reply:
I recommend just a single entry for these design that are being spread over multiple documents--just pointing to the top-level README for them.

Reply:
I think it would make sense to have just a single entry for Statics in Interfaces. The top level Statics in Interfaces doc can have links to the other ones.

Reply:
I would should for a process that can be easily generated with @omajid's update script. Leaving designs out is probably harder than, say, a prefix notation.

Reply:
The script uses a heuristic to separate main designs from subdesigns: If the document lists a PM/Dev (soon Owner), then the document is treated as a main (or top-level) design. 

For Statics In Interfaces, it sounds like the main design is `accepted/2021/statics-in-interfaces/README.md` and the rest are the components. If we removed PM/Dev from those documents, the scripts will just ignore them.

(I also think it makes sense to do anyway: there aren't separate Owners for the Design vs Requirements vs Scenarios, are there?)

Reply:
Alternatively, I can fix the script to ignore a certain level of nesting as sub-designs and only look for top-level documents.